packages/seller-client/src/components/RelativeDate/RelativeDate.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { formatDistanceToNow } from 'date-fns';
+import { Typography, Box } from '@material-ui/core';
+import TodayIcon from '@material-ui/icons/Today';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  icon: {
+    marginRight: spacing(1),
+  },
+}));
+
+type Props = {
+  date: Date;
+  prefix?: string;
+  showIcon?: boolean;
+};
+
+const formatDate = (date: Date) => {
+  const distanceToNow = formatDistanceToNow(date);
+  if (distanceToNow.includes('minute') || distanceToNow.includes('hour')) {
+    return 'Today';
+  } else if (
+    distanceToNow.includes('day') &&
+    +distanceToNow.split(' ')[0] < 7
+  ) {
+    return date.toDateString();
+  }
+  return `${date.getDate()}/${date.getMonth()}/${date.getFullYear()}`;
+};
+
+const RelativeDate: React.FC<Props> = ({ showIcon, prefix, date }) => {
+  const classes = useStyles();
+  const formattedDate = formatDate(date);
+  return (
+    <Box display="flex" alignItems="center">
+      {showIcon && <TodayIcon className={classes.icon} />}
+      <Typography variant="subtitle2">
+        {prefix ? (
+          <>
+            {`${prefix} `}
```

Comment:
I think this way is a little more direct

```suggestion
            {prefix}{' '}
```